superset/charts/data/api.py

```diff
@@ -358,7 +359,8 @@ def _send_chart_response(
         # This is needed for sending reports based on text charts that do the
         # post-processing of data, eg, the pivot table.
         if result_type == ChartDataResultType.POST_PROCESSED:
-            result = apply_post_process(result, form_data, datasource)
+            with g.telemetry("Post processing data"):
```

Comment:
I'm not super happy with the name `telemetry` here and the use of Flask `g` (though we do need to use a per-request singleton for this to work). Open to suggestions.